python/ray/data/_internal/planner/exchange/vanilla_shuffle_scheduler.py

```diff
@@ -0,0 +1,100 @@
+from typing import Any, Dict, List, Optional, Tuple
+
+from ray.data._internal.execution.interfaces import RefBundle
+from ray.data._internal.planner.exchange.interfaces import ExchangeScheduler
+from ray.data._internal.progress_bar import ProgressBar
+from ray.data._internal.remote_fn import cached_remote_fn
+from ray.data._internal.stats import StatsDict
+
+
+class VanillaShuffleScheduler(ExchangeScheduler):
```

Comment:
How about `PullBasedShuffleScheduler` or `SimpleShuffleScheduler`?

Reply:
I feel `Simple` is too general and hard to reason about. I am actually voting for `BulkShuffleScheduler` or `MapReduceShuffleScheduler`.

Reply:
cc @ericl and @jianoaix. Let's all decide a good naming here?

Reply:
I'm fine with pull-based or simple. Maybe pull-based is good since it aligns with push-based naming?

Reply:
Renamed to `PullBasedShuffleTaskScheduler`.